src/Notification/Notification.less

```diff
@@ -61,6 +61,9 @@
 		font-weight: @moon-notification-font-weight;
 		font-style: @moon-notification-font-style;
 		font-size: @moon-notification-font-size;
+		.enyo-locale-non-latin & {
+			font-family: @moon-non-latin-font-family;
```

Comment:
We gathering enyo-locale-non-latin on moonstone-text.less file. I recommand to move this into that just like what moon-body-text did.